### PR TITLE
Add examples to bottom of homepage

### PR DESCRIPTION
**Pull request by lambtron**
What do y'all think about this? Simply wanted to link to examples.deno.com somewhere on the home page. Open to ideas.

https://github.com/denoland/dotland/issues/2005

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -207,6 +207,47 @@ serve((req) => new Response("Hello World\\n"), { port: 8000 });`;
...
+            <h3 class="font-bold text-xl" id="examples">
+              Examples
+            </h3>
+          </a>
+          <p class="my-4 text-gray-700">
+            Here are some examples that you can use to get started immediately.
+          </p>
+          <ol class="ml-8 list-disc text-gray-700">
+            <li>
+              <a href="https://examples.deno.land/hello-world/" class="link">
```
Comment (lucacasonato) [inline pages/index.tsx:221]: Remove the trailing slash in the URL. With the trailing slash it 404's (I'll fix this upstream).

**Review by lucacasonato — APPROVED** [review]
LGTM!

**Comment by lucacasonato** [comment]
Just run `deno fmt` locally to get the CI to pass.

**Review by kt3k — APPROVED** [review]

**Comment by lambtron** [comment]
Hmm looks like there's an issue with the CLA. Signed it a few times now but still says "pending".

**Comment by satyarohith** [comment]
@lambtron, set your git user details:
```
$ git config --global user.name "John Doe"
$ git config --global user.email "john@doe.org" # use the email that you signed up with GitHub
```
And then push a commit:
```
$ git commit -m 'reset CI' --allow-empty
$ git push origin add-examples-to-homepage
```

if that doesn't work, we can merge without the license check and resolve the issue later.

**Pull request merged**